src/main/java/hu/ak/generics/cafeloop/listener/DatabaseInitListener.java

```diff
@@ -153,14 +157,48 @@ public void contextInitialized(ServletContextEvent sce) {
                 .name("Robusta Harmony")
                 .price(2750)
                 .description("Ízében a friss trópusi gyümölcsök édes tónusai dominálnak tejcsokoládéval és nádcukorral kombinálva.")
-                .imagePath("coffee-sample.jpg")
+                .imagePath("coffee-sample-3.jpeg")
                 .build();
 
         robustaHarmony = productDao.create(robustaHarmony);
 
+        createDummySubscription("Budapest, Fraknó utca 99.", monthly, testElek, Map.of(robustaFantasy, 3), subscriptionDao, subscriptionItemDao);
+        createDummySubscription("Budapest, Királyfürdő utca 128.", everyOtherWeek, testElek, Map.of(robustaFantasy, 10), subscriptionDao, subscriptionItemDao);
+        createDummySubscription("Budapest, Fraknó utca 99.", twiceAWeek, testElek, Map.of(arabicaElite, 1, robustaHarmony, 5), subscriptionDao, subscriptionItemDao);
+
```

Comment:
Most már létrehozhatunk elődefiniált előfizetéseket is. A metódust lásd lejjebb.